sdk/cosmos/azure-cosmos/azure/cosmos/_partition_health_tracker.py

```diff
@@ -181,6 +179,7 @@ def check_stale_partition_info(
         if current_time - self.last_refresh > REFRESH_INTERVAL:
```

Comment:
Please add units at the end of constant variable name such as `_MS` if the unit was milliseconds. Please update all other constatns wtih the same pattern. This will give better readability.
```suggestion
        if current_time - self.last_refresh > REFRESH_INTERVAL_MS:
```